fineract-provider/config/swagger/config.json.template

```diff
@@ -5,6 +5,7 @@
    "invokerPackage": "org.apache.fineract.client",
    "modelPackage": "org.apache.fineract.client.models",
    "apiPackage": "org.apache.fineract.client.services",
+   "artifactDescription": "A Java and Retrofit2 based client library for Apache Fineract 1.x Platform",
```

Comment:
I think we usually just call it "Apache Fineract" (VS "Fineract CN"), not "1.x" (I think).

```suggestion
   "artifactDescription": "A Java and Retrofit2 based client library for Apache Fineract Platform",
```